### DISCUSSION TITLE
Question : How to add Multiple Google Oauth ClientId and ClientSecret

### DISCUSSION THREAD

**Discussion by AdityaThakor67** [supabase/supabase#19674] [Questions]
Hello , I'm working on Project that support in multiple device like web , android and also ios , so the problem is google cloud console provides
different credential for different platform and i'm using supabase auth providers. In Auth Providers , Google required client ID & secret and i can add only one credential at time so is there any way in Auth Providers that i can enter multiple credentials in Google Auth in Supabase.

**Comment by GaryAustin1** [comment]
This is the last I see on it in gotrue repository... https://github.com/supabase/gotrue/issues/523

Also a more recent open issue on it: https://github.com/supabase/gotrue/issues/1347
